core/permission/useraccess.go

```diff
@@ -9,6 +9,11 @@ import (
 	"github.com/juju/names/v5"
 )
 
+// TODO hml 2024-02-05
```

Comment:
Should this get added to the risk register instead where we can properly track it?

Reply:
yes, I might make a jira ticket as well.